mutable_tree.go

```diff
@@ -427,6 +444,7 @@ func (tree *MutableTree) SaveVersion() ([]byte, int64, error) {
 // DeleteVersion deletes a tree version from disk. The version can then no
 // longer be accessed.
 func (tree *MutableTree) DeleteVersion(version int64) error {
+	debug("DELETE VERSION: %d\n", version)
```

Comment:
```suggestion
```

Reply:
There are debug statements in multiple parts of the code including `SaveVersion`. They will not print anything unless `debugging` is set to `true`

Reply:
Should we use a logging library here? we can then set logging level

Reply:
can you make an issue with this, we shouldn't expand the scope of this PR into logging as well

Reply:
Added #192